src/classes/atomTypeMix.cpp

```diff
@@ -194,19 +183,19 @@ std::pair<int, int> AtomTypeMix::indexOf(const std::shared_ptr<AtomType> &at1, c
             if (index == -1)
                 index = count;
             else
-                return {count, index};
+                return std::pair<int, int>{count, index};
```

Comment:
```suggestion
                return {{count, index}};
```
Learned this new trick today

Reply:
At last!  This has always bothered me.